### ISSUE TITLE
Scala dependencies don't link to repositories

### ISSUE THREAD

**Issue by kubukoz** [scalacenter/sbt-dependency-submission#48]
Hi! This is more of a question than an issue (perhaps you can enable Discussions in this repo?), but here it is:

I enabled the action on a couple repositories, but for any Scala dependencies I see this:

<img width="1475" alt="image" src="https://user-images.githubusercontent.com/894884/180574436-6a3ba06a-ba66-4cd6-be1f-cbdf2be053d6.png">

What can be done about that? Do the authors of the dependencies themselves need to take action?

**Comment by ckipp01** [comment]
> What can be done about that? Do the authors of the dependencies themselves need to take action?

No, the authors don't need to do anything. It's a bit of a mystery to me how they actually generate these. You'll also notice that some of the Java ones are incorrect. There is a discussion about this in https://github.com/community/community/discussions/19492. Hopefully it'll be addressed because getting Scala stuff linked would be amazing.
